src/mixins/date-mixin.ts

```diff
@@ -203,7 +203,7 @@ export default class DateMixin extends Vue {
     const yearDiff = (12 * (expiryDate.getFullYear() - currDate.getFullYear()))
     let difference = monthDiff + yearDiff
     // For example if Jan 31st and March 1, difference is 1 not 2
-    if (expiryDate.getDate() < currDate.getDate()) {
+    if ((expiryDate.getDate() + 25) < currDate.getDate()) {
```

Comment:
Was this just for testing?

Reply:
This was the small fix that I mentioned before. It's in the create-ui repo already but not in the shared components repo. I just wanted to add it.

Reply:
I don't understand the fix. Please add a code comment.